tutorials-and-examples/inference-servers/jetstream/http-server/Dockerfile

```diff
@@ -15,10 +15,12 @@ RUN apt -y update && apt install -y --no-install-recommends \
 RUN update-alternatives --install \
     /usr/bin/python3 python3 /usr/bin/python3.10 1
 
-RUN git clone https://github.com/google/JetStream.git && \
-cd /JetStream && \
-git checkout ${JETSTREAM_VERSION} && \
-pip install -e .
+RUN git clone https://github.com/google/jetstream-pytorch.git && \
+cd /jetstream-pytorch && \
+git checkout ${PYTORCH_JETSTREAM_VERSION} && \
+bash install_everything.sh
```

Comment:
Ditto, we shouldn't change the http server solely off of jetstream-pytorch version